linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_disable_nullok/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Disallow blank or null passwords in the PAM password-auth file'
+
+description: |-
+    Remove any instances of the <tt>nullok</tt> option in the
+    <tt>/etc/pam.d/password-auth</tt> file to prevent logons with empty passwords.
+    Note: Manual changes to the listed file may be overwritten by the
+    <tt>authselect</tt> program.
+
+rationale: |-
+    If an account has an empty password, anyone could log on and run commands
+    with the privileges of that account.
+    Accounts with empty passwords should never be used in operational environments.
+
+severity: high
+
```

Comment:
The tests fail because you have no CCE here.

```
 196/280 Test #197: missing-cces-rhel8 ....................................................***Failed    1.94 sec
The following rules in ssg-rhel8-ds.xml are missing CCEs:
xccdf_org.ssgproject.content_rule_accounts_passwords_pam_disable_nullok
```

You need to add identifiers key with a CCE entry. For example:

```
identifiers:
    cce@rhel8: CCE-12345-6
```

Get the CCE number in `shared/references/cce-redhat-avail.txt`. Pick a random number from `shared/references/cce-redhat-avail.txt`, use it as CCE ID in this rule and remove it from that file in the same commit.